### ISSUE TITLE
intraday mis stop loss order gets rejected 

### ISSUE THREAD

**Issue by rjamitsharma01** [krishnavelu/alice_blue#92]
INTRADAY MIS SLM gets rejected showing Rejection Reason : "Invalid price type"
while limit order and market buy order sent with this stoploss order gets executed

order code :

print(
       alice.place_order(transaction_type = TransactionType.Sell,
                     instrument = alice.get_instrument_by_symbol('NSE', "ZEEL"),
                     quantity = 168,
                     order_type = OrderType.StopLossMarket,
                     product_type = ProductType.Intraday,
                     price = 0.0,
                     trigger_price = 166.5,
                     stop_loss = None,
                     square_off = None,
                     trailing_sl = None,
                     is_amo = False)
Order status on python and alice blue app :
 "
![WhatsApp Image 2020-04-23 at 10 52 55 PM](https://user-images.githubusercontent.com/64218866/80130123-bf1ac980-85b5-11ea-8ad3-3c79c622ec47.jpeg)
{'status': 'success', 'message': 'Order placed successfully', 'data': {'oms_order_id': '200423000062231'}}"
I have tried triger price to "float(188)
but even on every try it shows the same issue only on "StopLossMarket"

please help

**Comment by hyndex** [comment]
Hello, Bro, i think this is due to
1. Your order type is StopLossMarket
2. Your stop-loss is stop_loss = None,..
Put the value of the stop-loss

**Comment by krishnavelu** [comment]
Try the latest version 0.0.7

**Comment by rjamitsharma01** [comment]
thanx latest version working fine
